File: Simple Regression: Two-Variable Regression Model/Two variable OLS Estimation.py
#!/usr/bin/env python
# coding: utf-8

# In[2]:


import pandas
import statsmodels.api as sm


# In[12]:


df = pandas.read_csv("C:/Users/User/Desktop/datasets/education-vs-wages.csv")
X = df['Years of Schooling']
Y = df['Hourly Wage']

X = sm.add_constant(X)
model = sm.OLS(Y, X).fit()
summary = model.summary()
print(summary)


# In[ ]:

File: Simple Regression: Two-Variable Regression Model/Two variable OLS Estimation.R
study <- read.csv("C:\\Users\\User\\Desktop\\STK 310 (Stats - regression analysis)\\section a\\code\\datasets\\education-vs-wages.csv")

summary(study)

x <- study$Years.of.Schooling
y <- study$Hourly.Wage

cov(x,y)
cor(x,y)

lrm <- lm(y ~ x, data = study)

coef(lrm)
summary(lrm)
vcov(lrm)

plot(x, y,
     main = "Scatter diagram of hourly wage againts years of schooling",
     pch = 20,
     col = "blue",
     xlab = 'Years of schooling',
     ylab = 'Hourly wage ($)')
abline(lrm, col = "midnightblue")
